DatabaseInitialScript.sql

```diff
@@ -169,6 +169,18 @@ IF NOT EXISTS (SELECT NULL FROM sys.tables WHERE NAME='OperationHistory')
   )
   GO
 
+  IF NOT EXISTS (SELECT NULL FROM sys.tables WHERE NAME='SystemErrors')
+    CREATE TABLE dbo.SystemErrors (
```

Comment:
For a system log table, it's not necessarily errors. It could anything like info, normal, important, critical.

If you change it a bit, you can use it for other purposes besides error